en/getting_started/README.md

```diff
@@ -0,0 +1,96 @@
+# Getting Started
+
+This topic explains how to get the *QGroundControl* source code and build it either natively or within a *Vagrant* environment. 
+It also provides information about optional or OS specific functionality.
+
+## Source Code
+
+Source code for *QGroundControl* is kept on GitHub here: https://github.com/mavlink/qgroundcontrol.
+It is [dual-licensed under Apache 2.0 and GPLv3](https://github.com/mavlink/qgroundcontrol/blob/master/COPYING.md).
+
+To get the source files:
+1. Clone the repo (or your fork):
+   ```
+   git clone --recursive https://github.com/mavlink/qgroundcontrol.git
+   ```
+2. Update submodules (do this each time you pull new source code):
+   ```
+   git submodule update
+   ```
+
+> **Tip** Github source-code zip files cannot be used because these do not contain the appropriate submodule source code. You must use git!
+
+
+## Build QGroundControl
+
+### Native Builds
+
+*QGroundControl* builds are supported for OSX, Linux, Windows, iOS and Android. *QGroundControl* uses [Qt](http://www.qt.io) as its cross-platform support library and uses [QtCreator](http://doc.qt.io/qtcreator/index.html) as its default build environment.
+
+- **OSX:** OSX 10.7 or higher, 64 bit, clang compiler
+  > **Warning** XCode 8 requires a workaround described below.
```

Comment:
This note was in the readme, but I can't find any workaround for XCode 8. Can you point me to it?

Reply:
Removed.